packages/vitest/src/node/reporters/benchmark/reporter.ts

```diff
@@ -56,7 +56,7 @@ export class BenchmarkReporter extends DefaultReporter {
     const duration = task.result.duration
 
     if (benches.length > 0 && benches.every(t => t.result?.state !== 'run' && t.result?.state !== 'queued')) {
-      let title = `\n ${getStateSymbol(task)} ${formatProjectName(task.file.projectName)}${getFullName(task, c.dim(' > '))}`
+      let title = `\n ${getStateSymbol(task)} ${formatProjectName(task.file.projectName, { colors: this.colors })}${getFullName(task, c.dim(' > '))}`
```

Comment:
Sounds like little cumbersome, I'm open for refactoring for something like `formatOptions`